printer_mods/mpfl/V0.1_skirt_OLED/README.md

```diff
@@ -0,0 +1,10 @@
+# V0.1 0.96" OLED skirt
+
+![Crappy photo](photo.jpg)
```

Comment:
Broken image, it's called png in your mod:

    ![Crappy photo](photo.png)

Reply:
Fixed in ffe868a